pkg/analyzer/detection/detection.go

```diff
@@ -99,10 +99,7 @@ func detectAndAnnotateCertifiedRHELComponents(name string, files analyzer.Files,
 			logrus.WithError(err).Errorf("failed to get RHCOS content sets for %v: %v", namespace, err)
```

Comment:
Can you update this error line to communicate we will continue with empty content sets?

Alternatively, move the `Warningf()` into this `if` block.